eclass/python-utils-r1.eclass

```diff
@@ -1321,6 +1321,14 @@ epytest() {
 		-Wdefault
 		# override color output
 		"--color=${color}"
+		# disable the undesirable-dependency plugins by default to
+		# trigger missing argument strips.  strip options that require
```

Comment:
Why 2 spaces here and on the next line?

Reply:
After the full stop? It's a an (older) style thing. I've never done it but some people learned it that way.

Reply:
Ah OK, thanks, it seems weird to me (just like [this](https://www.python.org/dev/peps/pep-0008/#inline-comments)).

Reply:
It makes spotting sentence boundaries easier.

Reply:
the yanks call is "french spacing" (eventhough IIRC France never did this). I also don't like it, but w/e